Formula/mtr.rb

```diff
@@ -48,6 +48,10 @@ def caveats
     <<~EOS
       mtr requires root privileges so you will need to run `sudo mtr`.
       You should be certain that you trust any software you grant root privileges.
+      Note: Default install will be in /usr/local/sbin. This will not be in your path by default. 
+        Options to fix in your bash / zsh profle config:
+          - alias mtr=‘/usr/local/sbin/mtr’
+          - export PATH="$PATH:/usr/local/sbin"
```

Comment:
`brew doctor` will already tell you to add sbin to your path. I don't see the value in adding this note.